libpod/container_inspect.go

```diff
@@ -97,11 +98,11 @@ func (c *Container) getContainerInspectData(size bool, driverData *inspect.Data)
 			SecondaryIPv6Addresses: nil,                    // TODO - do we support this?
 			EndpointID:             "",                     // TODO - is this even relevant?
 			Gateway:                "",                     // TODO
-			GlobalIPv6Addresses:    []string{},             // TODO - do we even support IPv6?
-			GlobalIPv6PrefixLen:    0,                      // TODO - do we even support IPv6?
-			IPAddress:              nil,
-			IPPrefixLen:            0,  // TODO
-			IPv6Gateway:            "", // TODO - do we even support IPv6?
+			GlobalIPv6Address:      "",
+			GlobalIPv6PrefixLen:    0,
+			IPAddress:              "",
+			IPPrefixLen:            0,
```

Comment:
We should be able to set this by `atoi` on `splitIP[1]`

Reply:
Hm sorry, I don't follow, it's getting set here: https://github.com/projectatomic/libpod/pull/953/files#diff-ff5abaa51a10c09feed1292fcf41380eR121

Reply:
Oh, oops.